<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="po_btnRemoveFilter">
       <property name="text">
        <string>Remove filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="po_btnEnableFilter">
       <property name="text">
        <string>Enable filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="po_le_NumberForbid"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
